cmd/influx/task_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/mock"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCmdTask(t *testing.T) {
+	orgID := influxdb.ID(9000)
+
+	fakeSVCFn := func(svc influxdb.TaskService) taskSVCsFn {
+		return func() (influxdb.TaskService, influxdb.OrganizationService, error) {
+			return svc, &mock.OrganizationService{
+				FindOrganizationF: func(ctx context.Context, filter influxdb.OrganizationFilter) (*influxdb.Organization, error) {
+					return &influxdb.Organization{ID: orgID, Name: "influxdata"}, nil
+				},
+			}, nil
+		}
+	}
+
+	t.Run("create", func(t *testing.T) {
+
+		//	checking cmd line tool gives all data needed for TaskService to actually create a Task
```

Comment:
Put this information (or a briefer version of it) in the test description instead of just `"create"` and remove this comment.